cargo check fails with things used from the image crate

This check was run on v0.3.0 because I was having trouble compiling my minimal test program, same thing happens on master branch

```
➜  rust_experiments git clone https://github.com/jerry73204/realsense-rust
Cloning into 'realsense-rust'...
remote: Enumerating objects: 418, done.
remote: Counting objects: 100% (418/418), done.
remote: Compressing objects: 100% (231/231), done.
remote: Total 418 (delta 271), reused 317 (delta 170), pack-reused 0
Receiving objects: 100% (418/418), 154.58 KiB | 904.00 KiB/s, done.
Resolving deltas: 100% (271/271), done.
➜  rust_experiments cd realsense-rust 
➜  realsense-rust git:(master) git checkout v0.3.0
Note: checking out 'v0.3.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 078f8e6 Let the device test gated by the device-test feature
➜  realsense-rust git:(078f8e6) cargo check
    Updating crates.io index
   Compiling autocfg v1.0.0
   Compiling libc v0.2.68
    Checking cfg-if v0.1.10
   Compiling proc-macro2 v1.0.10
   Compiling unicode-xid v0.2.0
   Compiling syn v1.0.17
   Compiling libm v0.2.1
    Checking lazy_static v1.4.0
   Compiling cc v1.0.50
   Compiling maybe-uninit v2.0.0
   Compiling byteorder v1.3.4
   Compiling getrandom v0.1.14
    Checking adler32 v1.0.4
   Compiling failure_derive v0.1.7
    Checking scopeguard v1.1.0
   Compiling memchr v2.3.3
   Compiling rustc-demangle v0.1.16
   Compiling proc-macro-nested v0.1.4
   Compiling rayon-core v1.7.0
   Compiling bitflags v1.2.1
   Compiling typenum v1.11.2
   Compiling crc32fast v1.2.0
    Checking futures-sink v0.3.4
    Checking futures-core v0.3.4
    Checking ppv-lite86 v0.2.6
   Compiling proc-macro-hack v0.5.15
    Checking pin-utils v0.1.0-alpha.4
    Checking slab v0.4.2
    Checking lzw v0.10.0
    Checking futures-io v0.3.4
    Checking futures-task v0.3.4
    Checking either v1.5.3
    Checking color_quant v1.0.1
    Checking rawpointer v0.2.1
    Checking bytemuck v1.2.0
    Checking scoped_threadpool v0.1.9
    Checking safe-transmute v0.11.0-rc.2
    Checking miniz_oxide v0.3.6
    Checking inflate v0.4.5
    Checking futures-channel v0.3.4
    Checking matrixmultiply v0.2.3
    Checking gif v0.10.3
   Compiling num-traits v0.2.11
   Compiling crossbeam-utils v0.7.2
   Compiling memoffset v0.5.4
   Compiling crossbeam-epoch v0.8.2
   Compiling num-integer v0.1.42
   Compiling num-complex v0.2.4
   Compiling num-rational v0.2.4
   Compiling num-iter v0.1.40
    Checking deflate v0.8.4
    Checking tiff v0.4.0
   Compiling quote v1.0.3
   Compiling backtrace-sys v0.1.35
    Checking png v0.16.2
    Checking crossbeam-queue v0.2.1
    Checking num_cpus v1.12.0
    Checking generic-array v0.13.2
    Checking rand_core v0.5.1
    Checking crossbeam-deque v0.7.3
    Checking rand_chacha v0.2.2
    Checking rand v0.7.3
    Checking approx v0.3.2
   Compiling backtrace v0.3.46
    Checking rayon v1.3.0
    Checking alga v0.9.3
    Checking rand_distr v0.2.2
    Checking nalgebra v0.19.0
    Checking jpeg-decoder v0.1.18
    Checking image v0.23.3
   Compiling synstructure v0.12.3
   Compiling futures-macro v0.3.4
   Compiling num-derive v0.3.0
   Compiling failure v0.1.7
    Checking futures-util v0.3.4
   Compiling realsense-sys v0.2.0 (/Users/rambo/devel/rust_experiments/realsense-rust/realsense-sys)
warning: unused imports: `BufReader`, `Path`, `collections::HashSet`, `fs::File`, `prelude::*`
 --> realsense-sys/build.rs:3:5
  |
3 |     collections::HashSet,
  |     ^^^^^^^^^^^^^^^^^^^^
4 |     fs::File,
  |     ^^^^^^^^
5 |     io::{prelude::*, BufReader},
  |          ^^^^^^^^^^  ^^^^^^^^^
6 |     path::{Path, PathBuf},
  |            ^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: method is never used: `to_string`
   --> realsense-sys/build.rs:156:5
    |
156 |     pub fn to_string(&self) -> String {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default

    Checking futures-executor v0.3.4
    Checking futures v0.3.4
    Checking realsense-rust v0.3.0 (/Users/rambo/devel/rust_experiments/realsense-rust)
error[E0432]: unresolved import `image::ConvertBuffer`
 --> src/base.rs:4:24
  |
4 | use image::{Bgr, Bgra, ConvertBuffer, DynamicImage, ImageBuffer, Luma, Rgb, Rgba};
  |                        ^^^^^^^^^^^^^ no `ConvertBuffer` in the root

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>` in the current scope
   --> src/base.rs:112:68
    |
112 |             Rs2Image::Bgr8(image) => DynamicImage::ImageBgr8(image.convert()),
    |                                                                    ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>` in the current scope
   --> src/base.rs:113:70
    |
113 |             Rs2Image::Bgra8(image) => DynamicImage::ImageBgra8(image.convert()),
    |                                                                      ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>` in the current scope
   --> src/base.rs:114:68
    |
114 |             Rs2Image::Rgb8(image) => DynamicImage::ImageRgb8(image.convert()),
    |                                                                    ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>` in the current scope
   --> src/base.rs:115:70
    |
115 |             Rs2Image::Rgba8(image) => DynamicImage::ImageRgba8(image.convert()),
    |                                                                      ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>` in the current scope
   --> src/base.rs:116:72
    |
116 |             Rs2Image::Luma16(image) => DynamicImage::ImageLuma16(image.convert()),
    |                                                                        ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use image::buffer_::ConvertBuffer;`

error: aborting due to 6 previous errors

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `realsense-rust`.

To learn more, run the command again with --verbose.
➜  realsense-rust git:(078f8e6) 
```

--- a/src/base.rs
+++ b/src/base.rs
@@ -2,7 +2,8 @@
 
 use crate::kind::{Format, StreamKind};
 #[cfg(feature = "with-image")]
-use image::{Bgr, Bgra, ConvertBuffer, DynamicImage, ImageBuffer, Luma, Rgb, Rgba};
+use image::buffer::ConvertBuffer;
+use image::{Bgr, Bgra, DynamicImage, ImageBuffer, Luma, Rgb, Rgba};
 #[cfg(feature = "with-nalgebra")]
 use nalgebra::{Isometry3, MatrixMN, Quaternion, Translation3, Unit, UnitQuaternion, Vector3, U3};
 use std::{

